SWEBackend/SWEBackend/API/RoomsAPI.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SWEBackend.Models;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using SWEBackend.Controllers.Rooms;
+
+namespace SWEBackend.API
+{
+    [ApiController]
+    [Route("api/rooms")]
+    public class RoomsAPI : ControllerBase
+    {
+        private readonly IRoomsController _roomsController;
+
+        public RoomsAPI(IRoomsController roomsController)
+            => _roomsController = roomsController;
+
+        [HttpGet]
+        public async Task<List<Room>> GetRoomsAsync(int? venueId)
+            => await _roomsController.GetRoomsAsync(venueId);
+
+        [HttpGet("{id}")]
+        public async Task<Room> GetRoomAsync(int id)
```

Comment:
I don't think that we should express ID's as ints

Reply:
auto-incremented int is the default identity column for sql, no need for UUIDs

Reply:
It's the default, but it probably isn't an awesome idea.
https://devforth.io/blog/why-your-software-should-use-uuids

Reply:
I just mean its definitely overkill for this project, but I can change it.